src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -129,9 +129,15 @@ bool CryptoManagerImpl::Init() {
   const SSL_METHOD* method;
 #endif
   switch (get_settings().security_manager_protocol_name()) {
+#ifdef OPENSSL_NO_SSL3
```

Comment:
@LuxoftAKutsan The commit has been amended with this change.

I will be taking over all of BTPankow's PR's in Core from now on, so you can ping me with any more feedback you have